### ISSUE TITLE
Can't figure out a way to do this ...

### ISSUE THREAD

**Issue by bananabr** [twisted/ldaptor#246]
Would it be possible to write an application that acts like a server/proxy in a way that, it accepts every bind request, binds itself to a backend server with a hardcoded credential, and just proxy requests and responses from that point on?

Thanks for all your help!

**Comment by cwaldbieser** [comment]
Yes.  You'd just need to intercept BIND requests and stop those from being proxied to the back end.  If you want the front end to be authenticated somehow-- you'd need to figure out what you are authenticating against and arrange that on a BIND request.  If the clients are never expected to BIND, you could just treat all BIND requests as failures.
